src/main/java/io/github/syst3ms/skriptparser/parsing/SyntaxParser.java

```diff
@@ -120,6 +121,22 @@
                 return variable;
             }
         }
+        // This is to prevent us from exchanging boolean operators with lists.
+        if (s.toLowerCase().startsWith("list ")) {
+            s = s.substring("list ".length());
+        } else {
+            // We parse boolean operators first to prevent clutter while parsing.
+            var operatorInfo = (ExpressionInfo<ExprBooleanOperators, Boolean>) SyntaxManager.getAllExpressions()
```

Comment:
This is worth caching, because we don't want a heavy Stream call to be repeated many times.